src/main/java/org/fundacionjala/coding/german/EANValidator.java

```diff
@@ -0,0 +1,54 @@
+package org.fundacionjala.coding.german;
+
+/**
+ * Created by Administrator on 8/29/2017.
+ */
+public class EANValidator {
+    /**
+     * @param number String
+     * @return boolean
+     */
+    public boolean validate(String number) {
+        boolean res = false;
+        int sumPar = sumPar(number);
+        int sumImpar = sumImpar(number);
+        int checkSum = sumImpar + sumPar;
+        if ((checkSum) % 10 == 0) {
+            if (number.charAt(12) == '0') {
+                res = true;
+            }
+
+        } else {
+            if ((10 - (checkSum % 10) == ((int) number.charAt(12) - 48))) {
```

Comment:
instead of using chars in ASCII code you can convert them to numeric values with Character.getNumericValue(char ch)